CHANGELOG.md

```diff
@@ -1,6 +1,29 @@
 Changelog
 =========
 
+## 5.0.0
```

Comment:
Perhaps a date here, too?

Reply:
I'm pretty much a "Just use git/rubygems for this information" kind of person, but two people have asked for it so I'll put it here as well.